ow_lander/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ add_action_files(
   FILES DigCircular.action
   FILES Grind.action
   FILES GuardedMove.action
+  FILES Antenna.action

```

Comment:
I meant all references.  
```suggestion
  FILES AntennaPanTilt.action
```

Reply:
Good catch @Samahu.  I'll agree.  While the action names technically aren't all verbs (they are nouns and/or verbs), they do specifically mention the action and not just the component.

Reply:
Done!